### PR TITLE
WIP:feat/metro

### PR DESCRIPTION
**Pull request by kenkus-futurice**

### CODE REVIEW

File: src/main/java/fi/hsl/transitdata/tripupdate/gtfsrt/GtfsRtFactory.java [hunk 1]
```diff
@@ -68,11 +69,14 @@ public static long lastModified(InternalMessages.StopEstimate estimate) {
     }
 
     public static GtfsRealtime.TripUpdate newTripUpdate(InternalMessages.StopEstimate estimate) {
-        final String routeName = reformatRouteName(estimate.getTripInfo().getRouteId());
         final int direction = PubtransFactory.joreDirectionToGtfsDirection(estimate.getTripInfo().getDirectionId());
+        String routeId = estimate.getTripInfo().getRouteId();
+        if (!ProcessorUtils.isMetroRoute(routeId)) {
+            routeId = reformatRouteName(routeId);
```
Comment (Divinare) [inline src/main/java/fi/hsl/transitdata/tripupdate/gtfsrt/GtfsRtFactory.java]: I guess the method name should be reformatRouteId

Comment (Divinare) [reply src/main/java/fi/hsl/transitdata/tripupdate/gtfsrt/GtfsRtFactory.java]: Btw why metroRouteId doesn't need reformatting? I also wonder why do you need to remove the last character if it is a number?

**Comment by Divinare** [comment]
LTGM, just one question and that method could be renamed (minor note).

File: src/main/java/fi/hsl/transitdata/tripupdate/gtfsrt/GtfsRtFactory.java (hunk 1 above)
Comment (kenkus-futurice) [reply src/main/java/fi/hsl/transitdata/tripupdate/gtfsrt/GtfsRtFactory.java]: It is a route variant. I don't know why it is removed.

**Pull request merged**